app/assets/javascripts/angular/controller/home-controller.js.erb

```diff
@@ -20,7 +20,11 @@ app.controller('HomeCtrl', ['$scope', '$http', '$rootScope', function($scope, $h
       $scope.card.bestActress = data.movie[0]["bestActress"];
       $scope.card.bestActorMovie = data.movie[0]["bestActorMovie"];
       $scope.card.bestActressMovie = data.movie[0]["bestActressMovie"];
-
+    })
+    $http.get('songs/' + year + '.json').success(function(data){
+      console.log(data.songs[0]['songlist']);
+      $scope.card.songlist = data.songs[0]['songlist'];
```

Comment:
`data.songs[0]['songlist']`

Is this by design where `data.songs` is an array? Will you be looping it in the future?